test/scripts/v1alpha3/check-katib-ready.sh

```diff
@@ -88,8 +88,32 @@ sed -i -e "s@gcr.io\/kubeflow-images-public\/katib\/v1alpha3\/suggestion-skopt@$
 
 cat manifests/v1alpha3/katib-controller/katib-config.yaml
 
+
+mkdir -p ${GO_DIR}
+cp -r . ${GO_DIR}/
+cp -r pkg/apis/manager/v1alpha3/python/* ${GO_DIR}/test/e2e/v1alpha3
+
+
+echo "Deploying tf-operator crds from kubeflow/manifests master"
```

Comment:
not sure if tf-operator installed here includes commit https://github.com/kubeflow/tf-operator/commit/4ee67e271527331a1bb0f00064ed80a090cf24ca, otherwise a TFJob may succeed but its trial will still keep running

Reply:
It is included. https://github.com/kubeflow/manifests/pull/397 is merged